sample-datapipeline-app/README.md

```diff
@@ -1,9 +1,42 @@
 # How to run
 
-## Requirements:
+## Option 1 - Locally using Python
+
+### Requirements
+
 - Python3
 - Install everything in requirements.txt
-  
+
+### Run
+
 1. Ensure that you have a .env file with appropriate environment variables as listed in the env-template.
 2. Run `python ehreceiver.py`
-3. Run `python ehsender.py`
\ No newline at end of file
+3. Run `python ehsender.py`
+
+## Option 2 - Using Kubernetes
+
+### Requirements
+
+- Kubernetes cluster
+
+### Run
+
+Use 'Helm install' command.
+Example:
+
+Navigate to the chart's directory and run:
+
+```bash

```

Comment:
Please add **"\\"** to end of line for single multi-line bash command, like..

helm install . \\
   --set env.EVENT_HUB_CONSUMER_GROUP="[value]" \\
   .... etc...

Reply:
👍